src/RestaurantPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+
+import { useDispatch } from 'react-redux';
+
+import { useParams } from 'react-router-dom';
+
+import {
+  selectRestaurant,
+  loadRestaurant,
+} from './actions';
+
+import RestaurantContainer from './RestaurantContainer';
+
+export default function RestaurantPage() {
+  const { id } = useParams();
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(selectRestaurant(id));
+    dispatch(loadRestaurant());
```

Comment:
```suggestion
    dispatch(loadRestaurant({ restaurantId }));
```

이렇게 하면 selectRestaurant라는 액션은 불필요해질 겁니다ㅎㅎ